### PR TITLE
Add details for parValue and maxNumberOfShares fields

### PR DESCRIPTION
**Pull request by eason-pan-bc**
*Issue #:* /bcgov/entity###  No ticket number for this

*Description of changes:*
As per discussion with Leo and Dave, made a quick update to add information about set relevant fields to `null` in certain scenarios, so that API users would be able to put correct value for the fields.

#### Before:
image

#### After:
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: docs/business.yaml [hunk 1]
```diff
@@ -6733,11 +6733,11 @@ components:
         maxNumberOfShares:
           type: number
           title: 'Maximum number of shares in the class. '
-          description: The maximum number of shares that a corporation can issue to the investors called authorized shares.
+          description: 'The maximum number of shares that a corporation can issue to the investors called authorized shares. If hasMaximumShares is false, set this field to null.'
         parValue:
           type: number
           title: 'Initial value of each share. '
-          description: 'The value of a single common share as set by a corporation''s charter. '
+          description: 'The value of a single common share as set by a corporation''s charter. If hasMaximumShares is false, set this field to null.'
```
Comment (AimeeGao) [inline docs/business.yaml:6740]: Good job, but for the description of parValue, should we change it to ' If hasParValue is false, set this field to null.'

Comment (eason-pan-bc) [reply docs/business.yaml:6740]: nice catch, pasted a wrong sentence from my notebook

Comment (eason-pan-bc) [reply docs/business.yaml]: fixed

**Review by AimeeGao — APPROVED** [review]
LGTM(^-^)

**Pull request merged**